src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorState.kt

```diff
@@ -0,0 +1,24 @@
+package zd.zero.waifu.motivator.plugin.settings
+
+class WaifuMotivatorState {
```

Comment:
So Kotlin and Lombok do not play nicely with each other at compile time. Rather than mess around with it, I just converted it to kotlin.

Thankfully Kotlin will generate the getters/setters for Java code, just like lombok. However, it does not do the `is` prefix, so `disableInDistractionFreeMode` had to be re-named.

So it is almost the same, it still works the same way though.

Reply:
Ohh, that explains the reason why other parts are changed. Do you think it'll take more effort if we separate the PR for converting the `WaifuMotivatorState` first to Kotlin and make this PR on top of that so that we can have a minimum change to the PR?

Reply:
If you wanted to test to make sure that that did not break any pre-existing thing. I can get that set up. I can do that, it is more effort but it is your project :grin: 

I can just rebase branch once the other changes are in.

Reply:
I dun did the rebase!